src/System.Private.Xml/tests/XmlNodeReader/System.Xml.XmlNodeReader.Tests/System.Xml.XmlNodeReader.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="$(CommonTestPath)\Tests.props" />
+  <PropertyGroup>
+    <ProjectGuid>{6C6F323D-054E-4723-9DEA-1C5E1E45DF88}</ProjectGuid>
```

Comment:
is this a new generated GUID or a copy & paste? If copy & paste please generate a new one (i.e. online GUID generator).